modules/installation-configuration-parameters.adoc

```diff
@@ -627,6 +627,12 @@ endif::openshift-origin[]
 |Specify one or more repositories that may also contain the same images.
 |Array of strings
 
+ifdef::aws[]
+|`platform.aws.lbType`
+|Required to set a load balancer type other than the default Classic, such as NLB. The ingress operator creates the load balancer for the default ingress controller based on the configuration used here.
```

Comment:
Required to set a load balancer type in AWS. Valid values are Classic or NLB.

Reply:
For other ingress controllers than the default if you don't Specify Classic or NLB in the ingress controller CR then the lbType which was set during install will referred. The installer sets the lbType value in ingress cluster config object.

Reply:
@miheer A value in this installation parameter is only truly required to set the NLB load balancer type, is that correct? The customer has the option to leave this field blank during installation and the installer will set lbType to Classic by default--am I understanding correctly? (Per the .yaml example.) Edits coming....TY for reviewing.